src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -11,4 +11,6 @@ public class Messages {
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
     public static final String MESSAGE_PERSON_FULL_VIEW = "Viewing %1$s in full view.";
     public static final String MESSAGE_PERSONS_LISTED_ACCORDING_TO_DAY = "Showing %1$d sessions";
+    public static final String MESSAGE_NAME_NOT_FOUND =
```

Comment:
would it be better if the messages were more consistent with the rest? Like maybe its enough to say that the name provided is invalid and we don't have to tell them to provide the correct name, since we didn't in the case of an incorrect index also

Reply:
hmm ok yea standardisation it is!